docs/androidx_test.md

```diff
@@ -14,104 +14,104 @@ It is now possible to use the AndroidX test runner in Robolectric tests. If you
 test runner, please check out the [configuration and plugin API][configuration-plugin-api], and let
 us know if there are any extension points missing that you require.
 
-**Robolectric**
+!!! snippet "Robolectric"
 
-=== "Java"
+    === "Java"
 
-    ```java
-    import org.robolectric.RobolectricTestRunner;
+        ```java
+        import org.robolectric.RobolectricTestRunner;
 
-    @RunWith(RobolectricTestRunner.class)
-    public class SandwichTest {
-    }
-    ```
+        @RunWith(RobolectricTestRunner.class)
+        public class SandwichTest {
+        }
+        ```
 
-=== "Kotlin"
+    === "Kotlin"
 
-    ```kotlin
-    import org.robolectric.RobolectricTestRunner
+        ```kotlin
+        import org.robolectric.RobolectricTestRunner
 
-    @RunWith(RobolectricTestRunner::class)
-    class SandwichTest
-    ```
+        @RunWith(RobolectricTestRunner::class)
+        class SandwichTest
+        ```
 
-**AndroidX Test**
+!!! snippet "AndroidX Test"
 
-=== "Java"
+    === "Java"
 
-    ```java
-    import androidx.test.ext.junit.runners.AndroidJUnit4;
+        ```java
+        import androidx.test.ext.junit.runners.AndroidJUnit4;
 
-    @RunWith(AndroidJUnit4.class)
-    public class SandwichTest {
-    }
-    ```
+        @RunWith(AndroidJUnit4.class)
+        public class SandwichTest {
+        }
+        ```
 
-=== "Kotlin"
+    === "Kotlin"
 
-    ```kotlin
-    import androidx.test.ext.junit.runners.AndroidJUnit4
+        ```kotlin
+        import androidx.test.ext.junit.runners.AndroidJUnit4
 
-    @RunWith(AndroidJUnit4::class)
-    class SandwichTest
-    ```
+        @RunWith(AndroidJUnit4::class)
+        class SandwichTest
+        ```
 
 ## Application
 
 Since most Android code is centric around a [`Context`][context-documentation], getting hold of your
 application’s context is a typical task for most tests.
 
-**Robolectric**
+!!! snippet "Robolectric"
```

Comment:
The issue here was that we used bold texts as a potential title.
Using h3 title didn't look too good either in my opinion.

I've tried the solution below. Let me know if you can think of an other approach.

<img width="723" alt="Screenshot 2024-10-27 at 17 47 05" src="https://github.com/user-attachments/assets/17efff1f-5877-4687-9da0-3d98664ce85f">